server/src/db/db.ts

```diff
@@ -0,0 +1,79 @@
+import Database from 'better-sqlite3'
+import type { PollingEvent, DateRecord } from "../dataTypes";
+
+const db = new Database('events.db', { verbose: console.log });
+
+export const createSchema = () => {
+db.exec(`
+    CREATE TABLE IF NOT EXISTS \`events\` (
+    \`id\` INTEGER PRIMARY KEY AUTOINCREMENT,
+    \`title\` VARCHAR(255) NULL,
+    \`location\` VARCHAR(512) NULL);
+
+    CREATE TABLE IF NOT EXISTS \`event_dates\` (
+    \`id\` INTEGER PRIMARY KEY AUTOINCREMENT,
+    \`timestamp\` INTEGER NOT NULL,
+    \`event_id\` INTEGER NOT NULL);
+
+    CREATE TABLE IF NOT EXISTS \`user_responses\` (
+    \`id\` INTEGER PRIMARY KEY AUTOINCREMENT,
+    \`user\` VARCHAR(512) NULL,
+    \`answer\` VARCHAR(45) NULL,
+    \`event_id\` INTEGER NOT NULL,
```

Comment:
event_id uz tu muze byt zbytecny/navic, protoze tu mas dates_id (coz by asi melo byt date_id - jednotne cislo). Ale ma to smysl mit, pokud by jsi si tim chtel ulehcit nejake dotazy a optimalizovat cteni. Takze jen zminuju, technicky tady zbytecne, ale muze se hodit do budoucna